### PR TITLE
added test coverage

### PR DESCRIPTION
**Pull request by olyode**
**Before creating a pull request make sure that:**

### JIRA link
https://tools.hmcts.net/jira/browse/RET-1054

### Change description
Added content to employee notice pay 
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/test/unit/views/notice-pay.test.ts [hunk 1]
```diff
@@ -16,8 +16,15 @@ const noticePayJson = JSON.parse(noticePayJsonRaw);
 const titleClass = 'govuk-heading-xl';
 const expectedTitle = noticePayJson.h1;
 const buttonId = 'main-form-submit';
-
+// const yesRadioButtonId = "noticePeriodLength";
```
Comment (lvo700) [inline src/test/unit/views/notice-pay.test.ts:19]: Can the commented lines be removed?

Comment (olyode) [reply src/test/unit/views/notice-pay.test.ts]: Yes removed

**Review by marcjudge — CHANGES_REQUESTED** [review]
Not a story in the sprint

**Comment by olyode** [comment]
Merged into 1023

**Pull request closed without merging**